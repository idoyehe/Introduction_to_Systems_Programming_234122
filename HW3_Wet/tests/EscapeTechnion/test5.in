


## day 0 ##

escaper add escaper1@gmail.com 13 8
escaper add escaper2@gmail.com 9 8
escaper add escaper3@gmail.com 9 8

company add bio1@gmail.com 10
company add med1@gmail.com 14
company add civil1@gmail.com 0

room add bio1@gmail.com 11 100 3 06-18 7
room add bio1@gmail.com 16 100 4 06-18 7
room add bio1@gmail.com 13 100 4 06-18 7

room add med1@gmail.com 21 100 3 06-18 7
room add med1@gmail.com 23 100 3 06-18 7

room add civil1@gmail.com 23 500 3 06-18 7

###should get room in med1@gmail.com
escaper recommend escaper1@gmail.com 3

###should get room in bio1@gmail.com
escaper recommend escaper2@gmail.com 3

escaper recommend escaper3@gmail.com 4
escaper recommend escaper2@gmail.com 3

escaper recommend escaper1@gmail.com 3
escaper recommend escaper1@gmail.com 3
escaper recommend escaper1@gmail.com 1

escaper order escaper3@gmail.com 0 23 0-15 2

report day
report best
report day
report best

### end ###