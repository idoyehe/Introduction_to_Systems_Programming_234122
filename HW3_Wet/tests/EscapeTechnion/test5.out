Day #0: 8 events in total
The order was made by escaper2@gmail.com, from the faculty of Chemistry, who is skilled at 8
	It was made for room 11, of bio1@gmail.com from Biology, for the time 6:00
	The room is best for a skill skill_level of 7. 3 people came for the total price of 300
The order was made by escaper3@gmail.com, from the faculty of Chemistry, who is skilled at 8
	It was made for room 13, of bio1@gmail.com from Biology, for the time 6:00
	The room is best for a skill skill_level of 7. 4 people came for the total price of 400
The order was made by escaper1@gmail.com, from the faculty of Computer Science, who is skilled at 8
	It was made for room 21, of med1@gmail.com from Medicine, for the time 6:00
	The room is best for a skill skill_level of 7. 3 people came for the total price of 300
The order was made by escaper2@gmail.com, from the faculty of Chemistry, who is skilled at 8
	It was made for room 11, of bio1@gmail.com from Biology, for the time 7:00
	The room is best for a skill skill_level of 7. 3 people came for the total price of 300
The order was made by escaper1@gmail.com, from the faculty of Computer Science, who is skilled at 8
	It was made for room 21, of med1@gmail.com from Medicine, for the time 7:00
	The room is best for a skill skill_level of 7. 3 people came for the total price of 300
The order was made by escaper1@gmail.com, from the faculty of Computer Science, who is skilled at 8
	It was made for room 21, of med1@gmail.com from Medicine, for the time 8:00
	The room is best for a skill skill_level of 7. 3 people came for the total price of 300
The order was made by escaper1@gmail.com, from the faculty of Computer Science, who is skilled at 8
	It was made for room 21, of med1@gmail.com from Medicine, for the time 9:00
	The room is best for a skill skill_level of 7. 1 people came for the total price of 100
The order was made by escaper3@gmail.com, from the faculty of Chemistry, who is skilled at 8
	It was made for room 23, of civil1@gmail.com from Civil and Environmental Engineering, for the time 15:00
	The room is best for a skill skill_level of 7. 2 people came for the total price of 1000
That is all for day #0!

The system has 18 faculties and in 1 days, they have made a revenue of 3000
One of the best faculties - Civil and Environmental Engineering, has made a revenue of 1000 so far!
One of the best faculties - Biology, has made a revenue of 1000 so far!
One of the best faculties - Medicine, has made a revenue of 1000 so far!

Day #1: 0 events in total
That is all for day #1!

The system has 18 faculties and in 2 days, they have made a revenue of 3000
One of the best faculties - Civil and Environmental Engineering, has made a revenue of 1000 so far!
One of the best faculties - Biology, has made a revenue of 1000 so far!
One of the best faculties - Medicine, has made a revenue of 1000 so far!

